### DISCUSSION TITLE
Enable sharing of history

### DISCUSSION THREAD

**Discussion by richardcase** [fidelity/kconnect#226] [Ideas]
Should we enable the sharing of history so that team members can share common cluster connection information?

For example, it could work something like this:

1. Someone connects to a bunch of clusters using `kconnect use` and sets aliases
2. The user exports their history (`kconnect history export --aliases cluster1,cluster2 -f export.yaml`)
3. File is shared with another user
4. The user import the history file (`kconnect history import -f export.yaml --user newusername --overwrite`)

When importing we'd need to overwrite the username or on export have an option to blank the username.

What do you think @rajarajanpsj @seankelly001?

**Comment by seankelly001** [comment]
Think this is a great idea. We also need to think about merging a users existing history with the imported one, and check for alias duplication/clashes etc

**Reply by richardcase** [reply]
Do you think we support merge & overwrite? And what if we use merge and an alias exists?
